metricbeat/module/mysql/_meta/config.reference.yml

```
@@ -2,6 +2,8 @@
   metricsets:
     - "status"
   #  - "galera_status"
+  #  - performance
+  #  - query
```

In the reference configuration we leave the default metricsets uncommented.
```suggestion
  metricsets:
    - "status"
    - performance
  #  - "galera_status"
  #  - query
```